bitcoin/BRWalletManager.h

```diff
@@ -326,6 +323,12 @@ BRWalletManagerCreateTransaction (BRWalletManager manager,
                                   const char *addr,
                                   uint64_t feePerKb);
 
+extern BRTransaction *
+BRWalletManagerCreateTransactionForSweep (BRWalletManager manager,
+                                          BRWallet *wallet,
+                                          BRWalletSweeper sweeper,
+                                          uint64_t feePerKb);
+
```

Comment:
The `sweeper` will include the UTXOs?

Reply:
Or will the BRWalletManager call out to BlockchainDB to accumulate them?

Reply:
`BRWalletSweeperHandleTransactionOutput` is how the sweeper gets populated with UTXOs.